deploy/upstream/0.1.0/02_catalogsourceconfig.crd.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Nit: Since the user on vanilla k8s will also install `OLM`, should we follow the same directory layout? Right now, OLM has this layout `deploy/upstream/manifests/{version}`?

Reply:
I think we can follow our own layout as long as we document it. 

For upstream, what we should we be doing is creating a `CatalogSource` with Marketplace in it. Add it to the OLM repo along with a `Subscription` so that we get installed automatically along with OLM. That way it is a one shot process. I am not saying this needs to be done now but can be done post 4.0.